azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/ParticipantInfoModel.kt

```diff
@@ -15,7 +15,7 @@ internal enum class ParticipantStatus {
 }
 
 internal data class ParticipantInfoModel(
-    val displayName: String,
+    var displayName: String,
```

Comment:
why mutable? 

You should be able to update the name with a copyOf constructor and a new instance of the ParticipantInfoModel. 

I see we have other Var's here too, but these classes should largely be immutable, especially if they are used in Redux state.

Reply:
As per discussion we will fix for GA.